thumbor/filters/format.py

```diff
@@ -11,17 +11,15 @@
 from thumbor.filters import BaseFilter, filter_method
 from thumbor.utils import logger
 
-
-ALLOWED_FORMATS = ['png', 'jpeg', 'jpg', 'gif', 'webp']
+ALLOWED_FORMATS = ["png", "jpeg", "jpg", "gif", "webp"]
 
 
 class Filter(BaseFilter):
-
     @filter_method(BaseFilter.String)
-    def format(self, format):
-        if format.lower() not in ALLOWED_FORMATS:
-            logger.debug('Format not allowed: %s' % format.lower())
+    async def format(self, file_format):
+        if file_format.lower() not in ALLOWED_FORMATS:
+            logger.debug(f"Format not allowed: {file_format.lower()}")
             self.context.request.format = None
         else:
-            logger.debug('Format specified: %s' % format.lower())
-            self.context.request.format = format.lower()
+            logger.debug(f"Format specified: {file_format.lower()}")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use % formatting in logging functions and pass the % parameters as arguments](https://app.codacy.com/manual/heynemann/thumbor/pullRequest?prid=4967909)